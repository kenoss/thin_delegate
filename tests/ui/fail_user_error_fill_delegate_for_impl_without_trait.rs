#[thin_delegate::register]
struct Hoge(String);

#[thin_delegate::fill_delegate]
impl Hoge {}

fn main() {}
