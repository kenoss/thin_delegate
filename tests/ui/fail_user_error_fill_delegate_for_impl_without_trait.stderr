error: expected `impl <Trait> for <Type>`
 --> tests/ui/fail_user_error_fill_delegate_for_impl_without_trait.rs:5:1
  |
5 | impl Hoge {}
  | ^^^^^^^^^^^^
